src/components/SelCard.js

```diff
@@ -0,0 +1,22 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import LinearProgressWithLabel from '@mui/material/LinearProgress'
+
+const SelCard = ({ title, description, progress }) => {
```

Comment:
should we pass color in as a parameter?

Reply:
agree with Jaana - can add color as a parameter and pass style into the Card component like `        <Card sx={{ maxWidth: 500 }} style={{ backgroundColor: color}}>
`